cmd/state/stateless/state.go

```diff
@@ -137,9 +166,14 @@ func StateGrowth1(chaindata string) {
 		}
 		return nil
 	})
-	check(err)
+	if err != nil {
+		r.db.Close()
```

Comment:
Could we use defer for it? It's really simple to forget about all this close and check

Reply:
Not sure if we need call `close` in cli command at all... 
I copy-pasted it from http handler...
Will think more on it tomorrow